spec/meta_parser_spec.rb

```diff
@@ -7,50 +7,52 @@
   let(:sample_fingerprint){ '66d8e9645db2878b53b6dd51acb672e97c389c87' }
   subject{ described_class.parse example_arg, groups_arg }
 
-  it "should extract the name and fingerprint of a test with no metadata" do
+  it 'should extract the name and fingerprint of a test with no metadata' do
     expect_result_options({
       name: 'Something should work',
       fingerprint: sample_fingerprint
     })
   end
 
-  describe "with multiple groups" do
+  describe 'with multiple groups' do
     let(:groups_arg){ [ group_double('A'), group_double('b'), group_double('c') ] }
     let(:sample_fingerprint){ '1c22e0ad110dc5b07761beec69712267fc384152' }
 
-    it "should build the name and fingerprint by concatenating all group descriptions and the example description" do
+    it 'should build the name and fingerprint by concatenating all group descriptions and the example description' do
       expect_result_options({
         name: 'A b c should work',
         fingerprint: sample_fingerprint
       })
     end
 
-    describe "with categories, tags and tickets" do
+    describe 'with categories, actives, tags and tickets' do
```

Comment:
Active as a noun is a grammatical term. Please replace mentions of "active" by "active flag" and "actives" by "active flags" in test descriptions.